lib/l10nUtil.js

```diff
@@ -12,6 +12,8 @@
 
 const path = require('path')
 const fs = require('fs')
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
```

Comment:
This can be combined to 1 line, just tested, seems to work.

Reply:
Fixed in cfc29b8a9f0729b44f2a45dec644ae29bf07e7e8